### ISSUE TITLE
Character Encoding Error

### ISSUE THREAD

**Issue by znnz** [djadmin/medium-cli#4]
Window 10
Powershell

Showing wrongly single quotes and double quotes, please have a look below image for reference
<img width="665" alt="medium_cli" src="https://cloud.githubusercontent.com/assets/6309215/15309086/170beec0-1c16-11e6-8bc9-8d1395aa5ff5.PNG">

**Comment by phocks** [comment]
Same problem here. Might be Windows related.

**Comment by djadmin** [comment]
This looks like Windows related bug. I'm using Mac OS X, and it's working fine.
![Image of Yaktocat](http://i.imgur.com/bxZT5Ls.png)

**Comment by znnz** [comment]
It's not window problem. It's string-stream problem. It's doen't encoded to utf-8. without using streaming it's work

**Comment by djadmin** [comment]
Yes, you are right. Can you please try out a quick fix I've placed @https://github.com/djadmin/medium-cli/commit/b9418d2532ccd70eb817081e97c30d30dd11106f and let me if issue still exists?

**Comment by znnz** [comment]
Nope. That doesn't seem to be solved  :grimacing:

**Comment by wkoffel** [comment]
I'm using Mac, Apple Terminal, and also have an encoding problem.

![](http://cl.ly/2E1q360o2f1Y/Screen%20Shot%202016-05-24%20at%2011.23.36%20AM.png "Medium CLI Encoding Errors")
